docs/reference/commandline/service_create.md

```diff
@@ -532,24 +532,23 @@ In this example, we are going to set the template of the created containers base
 service's name and the node's ID where it sits.
 
 ```bash
-$ docker service create --name hosttempl --hostname={% raw %}"{{.Node.ID}}-{{.Service.Name}}"{% endraw %} busybox top
```

Comment:
This is required for `jekyll` in the documentation… should not be removed :sweat: